roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -98,6 +98,11 @@
   failed_when: false
   when: not openshift.common.is_containerized | bool
 
+# Works around a 3.6 bug that left checkpoint files around
+- name: Clean up dockershim data
+  command: rm -rf /var/lib/dockershim/*
```

Comment:
/var/lib/dockershim/sandbox/*

Reply:
For my own personal edification, why not use the files module and state=absent?

Reply:
updated